### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by newhoteng**
### To Do list: add & remove
In this milestone:
- Add task function added
- Delete task function added
- Edit task function added
- Local storage is used to store task objects

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @newhoteng,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### These should be links, not just texts, to related sections in the readme file

dids

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/modules/crud.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+//     <span class="material-icons">more_vert</span>
+//   </div>
+//   <div class="bin">
+//     <span class="material-symbols-outlined delete-bin">delete</span>
+//   </div>
+//   `;
+
+//   ul.appendChild(taskItem);
+// };
+
```
Comment (Owusu-Desmond) [inline src/modules/crud.js]: - [x] Please make sure you remove commented code from your project to make your code clean. :pray: You should not push commented code to your repository. You can read more about it [here](https://www.freecodecamp.org/news/code-comments-the-good-the-bad-and-the-ugly-be9cc65fbf83/)

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @newhoteng , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

- [x] Why don't you give credit to everyone who inspired your codebase under the acknowledgment section of your readme?

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/crud.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+/* eslint-disable import/no-cycle */
+import ul from '../index.js';
```
Comment (Wes-Isaac) [inline src/modules/crud.js:2]: - [ ] [_OPTIONAL_] this is optional but please don't disable linters, if issues come up let's try to fix them.

**Review by Wes-Isaac — APPROVED** [review]
Hi @newhoteng, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professionally documented your work 👍.
- Use of Gitflow 💯.
- Great communication with reviewers 🥇.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**